drivers/serial/uart_nrfx_uart.c

```diff
@@ -16,6 +16,28 @@
 static NRF_UART_Type *const uart0_addr =
 		(NRF_UART_Type *)DT_NORDIC_NRF_UART_UART_0_BASE_ADDRESS;
 
+/* Device data structure */
+struct uart_nrfx_data {
+	struct uart_config dev_cfg;
```

Comment:
I think it would be good to name this field `uart_config` to avoid confusion with the value returned by `get_dev_config()`.